parser_library/src/workspace_manager_impl.h

```diff
@@ -262,6 +262,21 @@ class workspace_manager::impl final : public diagnosable_impl
         return make_continuous_sequence(file_manager_.get_virtual_file(id));
     }
 
+    continuous_sequence<opcode_suggestion> make_opcode_suggestion(
+        const std::string& document_uri, const std::string& opcode, bool extended)
+    {
+        auto suggestions =
+            ws_path_match(document_uri)
+                .make_opcode_suggestion(utils::resource::resource_location(document_uri), opcode, extended);
+
+        std::vector<opcode_suggestion> res;
+
+        for (auto&& [suggestion, distance] : suggestions)
+            res.emplace_back(opcode_suggestion { make_continuous_sequence(std::move(suggestion)), distance });
```

Comment:
This comes from SonarCloud:

```suggestion
            res.emplace_back(make_continuous_sequence(std::move(suggestion)), distance);
```

Reply:
no go - clang 12-14 does not support aggregate_type(...) construction.